config/routes.rb

```diff
@@ -3,7 +3,11 @@
 
   devise_for :users
 
-  resources :papers
+  resources :papers do
+    collection do
+      get 'pubmed_search'
```

Comment:
I think in the long run we'll integrate this into one search, so maybe just name it search?